File: fundamentos/operadores.py
def operadores():
    # Declaración de Variable
    edad = 18
    nacionalidad_mexicana = False

    # AND
    if edad >= 18 and nacionalidad_mexicana:
        print('Puede entregarse la INE')
    else:
        print('No se podrá entregar la INE')
        
    # OR
    if edad >= 18 or nacionalidad_mexicana:
        print('Puede pasar al antro')
    else:
        print('No puede pasar al antro')
    
    # NOT
    if not (edad >=18 or nacionalidad_mexicana):
        print('No puede pasar al antro')
    else:
        print('Puede pasar al antro')
        
operadores()

File: fundamentos/operadores.js
function operadores() {

    var edad = 18;
    var nacionalidadMexicana = false;

    // AND
    if (edad >= 18 && nacionalidadMexicana) {
        console.log('Puede entregarse la INE');
    } else {
        console.log('No se podrá entregar la INE');
    }

    // OR

    if (edad >= 18 || nacionalidadMexicana) {
        console.log('Puede pasar al antro');
    } else {
        console.log('No puede pasar al antro')
    }

    // NOT

    if (!(edad >= 18 || nacionalidadMexicana)) {
        console.log('No puede pasar al antro');
    } else {
        // si tienes 18 años o más ó es mexicano
        console.log('Puede pasar al antro');
    }

}
